app/experimenter/experiments/tests/test_models.py

```diff
@@ -1058,6 +1058,22 @@ def test_firefox_min_version_integer_returns_correct_integer(self):
 
         self.assertEqual(experiment.firefox_min_version_integer, 57)
 
+    def test_is_branched_addon_returns_true_for_addon_and_greater_version(self):
+        experiment = ExperimentFactory(
+            type=Experiment.TYPE_ADDON, firefox_min_version="70.0"
+        )
+        self.assertTrue(experiment.is_branched_addon)
+
+    def test_is_branched_addon_when_exp_is_addon_but_lower_version(self):
```

Comment:
I usually like to put both the expected behaviour/output and the test conditions in the name, but it's also hard to fit in a single line.  So here I would just include something about expecting it to be false when the version is lower.